### PR TITLE
feat (refs DPLAN-3428 AB#4195): Extend video player for embedded videos

### PR DESCRIPTION
**Pull request by gruenbergerdemos**
Ticket: [DPLAN-3428](https://demoseurope.youtrack.cloud/issue/DPLAN-3428)

Extend video player so it can also use embedded videos like from YouTube.

### CODE REVIEW

File: src/components/DpVideoPlayer/DpVideoPlayer.vue [hunk 1]
```diff
@@ -37,6 +47,12 @@ export default {
       default: false
     },
 
+    dataCy: {
+      type: String,
+      required: false,
+      default: 'videoplayer'
```
Comment (ahmad-demos) [inline src/components/DpVideoPlayer/DpVideoPlayer.vue:53]: videoPlayer*

Comment (gruenbergerdemos) [reply src/components/DpVideoPlayer/DpVideoPlayer.vue]: Done in [1fe4d4a](https://github.com/demos-europe/demosplan-ui/pull/1005/commits/1fe4d4a2745b43094c07226874423d294d465995)

**Review by elmasdemos — APPROVED** [review]
lgtm, code review

**Review by spiess-demos — COMMENTED** [review]
Code wise lgtm - cool improvement! I'd be glad to see this documented as a further example in DpVideoPlayer.stories.mdx.

**Comment by gruenbergerdemos** [comment]
> I'd be glad to see this documented as a further example in DpVideoPlayer.stories.mdx.
Thanks for the reminder! I'll update it in another PR, so this can be tested for the sprint. 

> And - will this be also implemented in main branch?
I think so, yes.

**Pull request merged**